### DISCUSSION TITLE
Add community-made Nuxt modules of Naive UI to the docs

### DISCUSSION THREAD

**Discussion by typed-sigterm** [tusen-ai/naive-ui#5655] [General]
- `@bg-dev/nuxt-naiveui`: https://github.com/becem-gharbi/nuxt-naiveui
- `naive-ui-nuxt`: https://github.com/kuizuo/naive-ui-nuxt (⚠️ not support Nuxt 3.10 currently)

**Comment by jahnli** [comment]
We provide official nuxt plugin now. Check https://github.com/07akioni/naive-ui-nuxt-demo again.
